addons/controllers/csi/vspherecsiconfig_controller.go

```diff
@@ -319,6 +326,10 @@ func (r *VSphereCSIConfigReconciler) ConfigMapToVSphereCSIConfig(o client.Object
 	_ = r.List(context.Background(), configs)
 	requests := []ctrl.Request{}
 	for i := 0; i < len(configs.Items); i++ {
+		// avoid enqueuing reconcile requests for template vSphereCSIConfig CRs in event handler of ConfigMap CR
+		if _, ok := configs.Items[i].Annotations[constants.TKGAnnotationTemplateConfig]; ok {
```

Comment:
**issue**: We should check for system namespace here too right ?

Reply:
Yeah! I missed that.. Updated